patterns/ecs-v1/exim

```diff
@@ -1,13 +1,25 @@
 EXIM_MSGID [0-9A-Za-z]{6}-[0-9A-Za-z]{6}-[0-9A-Za-z]{2}
-EXIM_FLAGS (<=|[-=>*]>|[*]{2}|==)
-EXIM_DATE %{YEAR:exim_year}-%{MONTHNUM:exim_month}-%{MONTHDAY:exim_day} %{TIME:exim_time}
-EXIM_PID \[%{POSINT}\]
+# <=     message arrival
+# =>     normal message delivery
+# ->     additional address in same delivery
+# *>     delivery suppressed by -N
+# **     delivery failed; address bounced
+# ==     delivery deferred; temporary problem
+EXIM_FLAGS (<=|=>|->|*>|**|==|<>|>>)
```

Comment:
do the literal asterisks (`*`) need escaping here?
does this grouping need a capture?

```suggestion
EXIM_FLAGS (?:<=|=>|->|\*>|\*\*|==|<>|>>)
```